### PR TITLE
Improved README.md

### PR DESCRIPTION
**Pull request by jakubpawlina**
- [x] Included a screenshot showing after the changes.
- [x] Included a description of the change below.
- [x] Related to some issue.

## Description
Improved README.md file:
- added links to repositories and ordered in points
- added screenshot from website
- added link to FOSSASIA website
- added "How to get geolocation" section
- added link to GCI18 Fossasia website
- updated description saying which website supports it no
- small fixes added

Fixes: #18 

## Screenshot after changes
zrzut ekranu 127

### CODE REVIEW

**Comment by abishekvashok** [comment]
I would be glad if you could relate this to some issue

**Comment by jakubpawlina** [comment]
@abishekvashok done. I have also expanded that for the remaining points

File: README.md [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
 ## How to use geolocation for other projects
-The website currently supports only the gci17.fossasia.org repository where you can add the geo co-ordinates for mentors and students, the
-members of FOSSASIA are free to supply any further integrations, check the code in the index.html and do the needy.
+The website currently supports only the [gci18.fossasia.org](https://github.com/fossasia/gci18.fossasia.org) repository where you can add the geo coordinates for mentors and students, the members of FOSSASIA are free to supply any further integrations, check the code in the <code>index.html</code> and do the needy.
+
+### How to get geolocation
+- visit [geolocation website](https://fossasia.github.io/geolocation/)
+- click "Get my geolocation" button
+- copy the coordinates and paste where wanted (you can add them directly to Mentors file \[If you are a mentor] in supported repository by visiting link in the bottom
+- it's everything, enjoy!
```
Comment (abishekvashok) [inline README.md]: Please remove this line

File: README.md [hunk 2]
```diff
@@ -1,14 +1,21 @@
 # Geolocation finder
 This website is the official website of Fossasia when it comes to geolocations. It finds geolocation for integration in multiple projects 
-the community looks after. This is basically a mini-project of FOSSASIA.
+the community looks after. This is basically a mini-project of [FOSSASIA](https://fossasia.org/).
+
+![zrzut ekranu 126](https://user-images.githubusercontent.com/29257057/49386524-09c94680-f720-11e8-86ee-c066c0a8652d.png)
```
Comment (abishekvashok) [inline README.md]: Please host this image directly on the repo and please use a good title.

**Comment by jakubpawlina** [comment]
@abishekvashok done

**Comment by abishekvashok** [comment]
Thanks

**Pull request merged**